Tutorials/Creating-and-distributing-a-package/index.md

```diff
@@ -0,0 +1,500 @@
+    <file>
+      <guid>PackageWorkshop.dll</guid>
+      <orgPath>~/bin</orgPath>
+      <orgName>PackageWorkshop.dll</orgName>
+    </file>
+  </files>
+  ...
+</umbPackage>
+```
+
+So here you will notice that while we only added the top level folder in the backoffice, it went through that folder and added a `<file>` element for every file found within that folder. And each file has 3 properties - `guid`, `orgPath` & `orgName`. 
+
+You don't have to worry too much about guid and orgName, they are references to the file name. Umbraco will automatically rename the file if there are conflicts, since they are stored in a flat structure it is quite likely, fx lets say you had these two files in your package:
+
+~/App_Plugins/PackageWorkshop/Dashboard/main.css
+~/App_Plugins/PackageWorkshop/ContentApp/main.css
+
+Both would have the same name and be thrown in the root of the zip file - this would cause a conflict and Umbraco would rename them to randomly generated names. These names would correspond to the `guid` property, while the `orgName` and `orgPath` would make up the original names - and would be where the package would place and name the file when being intalled.
+
+## Creating a draft package on Our
+
+At this point we have a complete package and can push it to [Our Umbraco](https://our.umbraco.com/packages/) which is the where the Umbraco package repository is. The packages on that website are also the ones that are featured in the package section of the backoffice.
+
+To create a package on Our, you first need an account on there and the account needs to have a certain level of karma - which can be gained by responding to forum posts.
+
+Once you log in you can go to your [package overview][member-package-overview] which should look a bit like this:
+
+![Package overview][package-overview]
+
+Click the "Add package" button and fill out all the information, upload the package and save in the end.
+
+:::warning 
+If you don't intend for people to use the package (as in this tutorial), then please don't click the "Go live" button in the end.
+:::
+
+Now your package is on Our, and if the "Go live" button is clicked it is visible for all to see! Next step is to make it a bit simpler to deploy updates to the package. It is perfectly fine to log in here, and upload a new version each time. The next steps will show an easier way though..
+
+## Pushing your package to Github
+
+If you are creating a package in order to share it with others it is a great idea to also share the source code. It is the open source way!
+
+To share it, and make it easier to manage and deploy updates we will set up a Github repository for the package. This tutorial assumes you know what Github is, and that you have an account. 
+
+Create a fresh repo, with no readme, gitignore nor based on a template. On the second screen it will give you a command to push an existing repository to the new Github repo, should look like this but with your own user in the link:
```

Comment:
wasn't sure what the nor text meant here..?

Reply:
ahh I understand... how about

Create a fresh repo, with no readme or gitignore - do not choose a repository template (set to 'No Template').